config/routes.rb

```diff
@@ -34,6 +34,15 @@
     resources :case_notes, only: [:show, :create, :update, :destroy] do
       patch 'internal', to: 'case_notes#internal', on: :member
     end
+
+    scope '/assignments' do
```

Comment:
You should replace this line with `resources :assignments, only: [:show, :create, :update, :destroy] do` like you had on line 45, and delete line 45

Reply:
I have this so the route ends up being `/assignments/templates` instead of `/templates`. Are you good with just `/templates` as the route?